hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionConfiguration.java

```diff
@@ -44,6 +51,23 @@
   public static final int PRIMARY_SCAN_TIMEOUT_MICROSECOND_DEFAULT = 1000000; // 1s
   public static final String LOG_SCANNER_ACTIVITY = "hbase.client.log.scanner.activity";
 
+  /**
+   * Parameter name for client pause when server is overloaded, denoted by
+   * {@link HBaseServerException#isServerOverloaded()}
+   */
+  public static final String HBASE_CLIENT_PAUSE_FOR_SERVER_OVERLOADED =
+    "hbase.client.pause.server.overloaded";
+
+  static {
+    // This is added where the configs are referenced. It may be too late to happen before
+    // any user _sets_ the old cqtbe config onto a Configuration option. So we still need
+    // to handle checking both properties in parsing below. The benefit of calling this is
+    // that it should still cause Configuration to log a warning if we do end up falling
+    // through to the old deprecated config.
+    Configuration.addDeprecation(
```

Comment:
Is there any problem from this deprecation being added twice?

Reply:
No this should be safe, based on the implementation of the call which is idempotent. But maybe from the perspective of my response to your other comment I'll just remove it for simplicity in master.

Reply:
This is moot now, as of latest commit